Latest code in master branch won't compile on CentOS

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
CentOS Linux release 7.5.1804
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
N/A
- TensorFlow installed from (source or binary):
Source
- TensorFlow version:
Latest code in master branch
- Python version:
2.7
- Installed using virtualenv? pip? conda?:
pip
- Bazel version (if compiling from source):
1.9
- GCC/Compiler version (if compiling from source):
6.3.0
- CUDA/cuDNN version:
N/A
- GPU model and memory:
N/A



**Describe the problem**
When compile with the following command line: 
bazel build -c dbg //tensorflow/tools/pip_package:build_pip_package
I got the following error:
external/eigen_archive/Eigen/src/Core/products/GeneralBlockPanelKernel.h: In member function 'void Eigen::internal::gebp_kernel<LhsScalar, RhsScalar, Index, DataMapper, mr, nr, ConjugateLhs, ConjugateRhs>::operator()(const DataMapper&, const LhsScalar*, const RhsScalar*, Index, Index, Index, Eigen::internal::gebp_kernel<LhsScalar, RhsScalar, Index, DataMapper, mr, nr, ConjugateLhs, ConjugateRhs>::ResScalar, Index, Index, Index, Index) [with LhsScalar = float; RhsScalar = float; Index = long int; DataMapper = Eigen::internal::blas_data_mapper<float, long int, 0, 0>; int mr = 8; int nr = 4; bool ConjugateLhs = false; bool ConjugateRhs = false]':
external/eigen_archive/Eigen/src/Core/products/GeneralBlockPanelKernel.h:1419:13: error: inconsistent operand constraints in an 'asm'
             EIGEN_GEBGP_ONESTEP(0);
             ^
and more similar errors.

**Provide the exact sequence of commands / steps that you executed before running into the problem**
see the commands above

**Any other info / logs**
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

--- a/tensorflow/workspace.bzl
+++ b/tensorflow/workspace.bzl
@@ -136,11 +136,11 @@ def tf_workspace(path_prefix = "", tf_repo_name = ""):
     tf_http_archive(
         name = "eigen_archive",
         build_file = clean_dep("//third_party:eigen.BUILD"),
-        sha256 = "753fbb58d0a49b6bcbcfb126ebfa2e21fc97f7471529ba835a096008ce588d8a",
-        strip_prefix = "eigen-eigen-9f48e814419e",
+        sha256 = "9de38f2d162c51599b802f7c36d9f3773980d19ac908c61638f8344d2c10e1ca",
+        strip_prefix = "eigen-eigen-88fc23324517",
         urls = [
-            "https://mirror.bazel.build/bitbucket.org/eigen/eigen/get/9f48e814419e.tar.gz",
-            "https://bitbucket.org/eigen/eigen/get/9f48e814419e.tar.gz",
+            "https://mirror.bazel.build/bitbucket.org/eigen/eigen/get/88fc23324517..tar.gz",
+            "https://bitbucket.org/eigen/eigen/get/88fc23324517.tar.gz",
         ],
     )
 

